demos/python_demos/instance_segmentation_demo/README.md

```diff
@@ -1,28 +1,34 @@
 # Instance Segmentation Python* Demo
 
-This demo shows how to run Instance Segmentation models from `Detectron` or `maskrcnn-benchmark` using OpenVINO&trade;.
-These models should be obtained through [OpenVINO&trade; Training Extensions (OTE)](https://github.com/openvinotoolkit/training_extensions/tree/develop/pytorch_toolkit/instance_segmentation#get-pretrained-models).
+This demo shows how to run Instance Segmentation models from [OpenVINO&trade; Training Extensions (OTE)](https://github.com/openvinotoolkit/training_extensions/tree/develop/pytorch_toolkit/instance_segmentation#get-pretrained-models) and `yolact` models family.
 
 > **NOTE**: Only batch size of 1 is supported.
 
 ## How It Works
 
 The demo application expects an instance segmentation model in the Intermediate Representation (IR) format with the following constraints:
-
-* Two inputs: `im_data` for input image and `im_info` for meta-information about the image (actual height, width and scale).
-* At least four outputs including:
-    * `boxes` with absolute bounding box coordinates of the input image
-    * `scores` with confidence scores for all bounding boxes
-    * `classes` with object class IDs for all bounding boxes
-    * `raw_masks` with fixed-size segmentation heat maps for all classes of all bounding boxes
+1. for instance segmentation models based on `Mask RCNN` approach:
+    * Two inputs: `im_data` for input image and `im_info` for meta-information about the image (actual height, width and scale).
+    * At least four outputs including:
+        * `boxes` with absolute bounding box coordinates of the input image
+        * `scores` with confidence scores for all bounding boxes
+        * `classes` with object class IDs for all bounding boxes
+        * `raw_masks` with fixed-size segmentation heat maps for all classes of all bounding boxes
+2. for instance segmentation models based on `YOLACT` approach:
+    * Single input for input image.
+    * At least four outputs including:
+      * `boxes` with normalized in [0, 1] range bounding box coordinates
```

Comment:
The indentation is inconsistent between the two alternatives.

Reply:
fixed